<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAutoMode</class>
 <widget class="QWidget" name="QAutoMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAutoMode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab0">
      <attribute name="title">
       <string>Notice</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;This function only work with Android Phone, which enable ADB Mode.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;See &lt;/span&gt;&lt;a href=&quot;https://developer.android.com/studio/command-line/adb&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#0000ff;&quot;&gt;reference&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::MarkdownText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Test</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pbTest">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labTest">
            <property name="text">
             <string>Not tested yet</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic Setting</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Total number</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbTotalNum">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Folder to save images</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leImagePath">
         <property name="text">
          <string>.\capture</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbTest</sender>
   <signal>clicked()</signal>
   <receiver>QAutoMode</receiver>
   <slot>runTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStart</sender>
   <signal>clicked()</signal>
   <receiver>QAutoMode</receiver>
   <slot>startCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>QAutoMode</receiver>
   <slot>tabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runTest()</slot>
  <slot>startCapture()</slot>
  <slot>tabChanged(int)</slot>
 </slots>
</ui>
